controllers/feedItemController.js

```diff
@@ -16,18 +16,19 @@ var FeedItemController = {
 
         if (requestedIsActive) {
             query = query.where('isActive').equals(requestedIsActive);
-        }
+         }
+
         if (requestedType) {
             query = query.where('type').equals(requestedType);
         }
         if (requestedIndex) {
-            query = query.where('index').lt(requestedIndex);
+            query = query.where('_id').lt(mongoose.Types.ObjectId(requestedIndex));
```

Comment:
I'm pretty shocked this works. The `index` param doesn't have to be a number any more, but `.lt` still works?

Reply:
yeah crazy right? but that ObjectID thing is a special object Mongo has that has a bunch of special features. Sorting is one of those: https://docs.mongodb.com/manual/reference/method/ObjectId/